frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -0,0 +1,333 @@
+                                    <td onClick={(e) => e.stopPropagation()}>
+                                        <input
+                                            type="checkbox"
+                                            className="checkbox checkbox-sm"
+                                            checked={isSelected}
+                                            onChange={() =>
+                                                handleToggleRow(section.id)
+                                            }
+                                        />
+                                    </td>
+                                    <td>{section.facility.name}</td>
+                                    <td>{section.facilitator}</td>
+                                    <td>
+                                        {new Date(
+                                            section.start_date
+                                        ).toLocaleDateString('en-US', {
+                                            year: 'numeric',
+                                            month: 'short',
+                                            day: 'numeric'
+                                        })}
+                                    </td>
+                                    <td>
+                                        {new Date(
+                                            section.end_date
+                                        ).toLocaleDateString('en-US', {
+                                            year: 'numeric',
+                                            month: 'short',
+                                            day: 'numeric'
+                                        })}
+                                    </td>
+                                    <td>
+                                        <div className="flex items-center">
+                                            <div className="w-24 h-2 bg-gray-300 rounded relative">
+                                                <div
+                                                    className="h-full bg-green-500 rounded"
+                                                    style={{
+                                                        width: `${(section.enrolled_residents / section.capacity) * 100}%`
+                                                    }}
+                                                ></div>
+                                            </div>
+                                            <span className="ml-2 text-sm">
+                                                {section.enrolled_residents}/
+                                                {section.capacity}
+                                            </span>
+                                        </div>
+                                    </td>
+                                </tr>
+                            );
+                        })}
+                    </tbody>
+                </table>
```

Comment:
@calisio Do you think we should create the Sections Table as a component or is it too soon to do something like that?